config/storageClasses.yaml

```diff
@@ -23,6 +23,14 @@ storageClasses:
     pytype: lsst.afw.cameraGeom.Camera
   Detector:
     pytype: lsst.afw.cameraGeom.Detector
+  Box2I:
+    pytype: lsst.geom.Box2I
+  Extent2I:
+    pytype: lsst.geom.Extent2I
+  Point2I:
+    pytype: lsst.geom.Point2I
+  Filter:
+    pytype: lsst.afw.image.Filter
```

Comment:
That's an interesting thought albeit not for this ticket. So for `StorageClass("lsst.afw.image.Filter")` then rather than defaulting to `object` we assume that the pytype name is the storage class name... (we could even adopt the convention that names that start with upper case are never pytypes). Worth pondering.